database/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM postgres
```

Comment:
Same as with Node. If there is no special reason for choosing full Debian i recommend looking into alpine. if there are not critical trade-offs which we cannot make i recommend using it instead.